packages/flutter_tools/lib/src/web/compile.dart

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:flutter_tools/src/build_system/targets/assets.dart';
```

Comment:
`import '../build_system/targets/assets.dart';`

but it looks like it's unused?

Reply:
Yeah I started web and it won't quite work yet - reverted this part.